handlers/external/external.go

```diff
@@ -344,13 +356,41 @@ func (h *handler) SendMsg(ctx context.Context, msg courier.Msg) (courier.MsgStat
 	return status, nil
 }
 
+func buildQuickRepliesResponse(quickReplies []string, sendMethod string, contentType string) string {
+	var response, format, separator string
+
+	switch {
+	case (sendMethod == http.MethodPost || sendMethod == http.MethodPut) && contentType == contentJSON:
+		response = "[]"
+		format = "[\"%s\"]"
+		separator = "\",\""
+
+	case (sendMethod == http.MethodPost || sendMethod == http.MethodPut) && contentType == contentXML:
+		response = ""
+		format = "<item>%s</item>"
+		separator = "</item><item>"
+
+	default:
+		response = ""
+		format = "%s"
+		separator = "\\,"
```

Comment:
is there a more natural separator than `\,`? just thinking that `x\,y` is a valid quick reply anyway

Reply:
Good point, maybe the appropriate way to do this is just have repeating values, ie:
  `text=foo&quick_reply=one&quick_reply=two`

Reply:
yeah was thinking that tho it means `{{ quick_replies }}` contains the key/values so you can't say so `qr=one&qr=two` but maybe that's ok

Reply:
I submitted some changes, but I kept the format with `\,`, what do you decide?

Reply:
And how would it be replaced in the send URL/body? Because the default is key-value pairs.

Reply:
The tricky part is that we don't restrict any characters in quick replies.. so no character(s) is strictly safe as a separator. So I think I'd lean toward it evaluating like...

`http://endpoint.com?{{quick_replies}}` 

=>

`http://endpoint.com?quick_reply=one&quick_reply=two`

Since that's the most correct way to send multiple values in a URL

Reply:
Great! And for the tests, by default is it allowed to compare multiple values ​​this way?

Reply:
not sure I understand... tests? 

@nicpottier how you feel about this?

Reply:
Yes, at `external_test.go`.